timed/employment/serializers.py

```diff
@@ -214,88 +228,176 @@ class Meta:
         ]
 
 
-class AbsenceTypeSerializer(ModelSerializer):
-    """Absence type serializer."""
+class UserAbsenceTypeSerializer(ModelSerializer):
+    """Absence type serializer for a user.
+
+    This is only a simulated relation to the user to show the absence credits
+    and balances.
+    """
+
+    credit        = SerializerMethodField()
+    used_duration = SerializerMethodField()
+    used_days     = SerializerMethodField()
+    balance       = SerializerMethodField()
+
+    absence_credits = SerializerMethodResourceRelatedField(
+        source='get_absence_credits',
+        model=models.AbsenceCredit,
+        many=True,
+        read_only=True
+    )
+
+    user = SerializerMethodResourceRelatedField(
```

Comment:
I think it would be better to have a ForeignKey on the UserAbsenceType proxy model then this could SerializerMethodResourceRelatedField would be obsolete.